frontend/src/types/Graph.ts

```diff
@@ -474,6 +474,8 @@ export interface DecoratedGraphNodeData extends GraphNodeData {
   httpOut: number;
   // true if has istio namespace
   isIstio?: boolean;
+  // true if is an Ambient Istio waypoint
+  isWaypoint?: boolean;
```

Comment:
This should actually be moved to the base type,`GraphNodeData`.    You may also want to add this in the `NodeAttr` definition.

Reply:
Moved and added also to NodeAttr